how-to-docs/drone-how-to.md

```diff
@@ -128,17 +128,15 @@ If your repository is hosted on Gitlab, you don't want to publish your images to
 Register for a free [Quay account](https://quay.io) using your Github account linked to the Home Office organisation.
 
 Once you've logged into Quay check that you have `ukhomeofficedigital` under Users and Organisations.  
-If you do not, [make a request by adding an issue to the Platform Access column](https://github.com/UKHomeOffice/application-container-platform-bau/projects/1)
+If you do not, [submit a support request on the platform hub for access to the ukhomeoffice organisation](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview).
```

Comment:
Is there a specific support request form you can link to here?

Reply:
It doesn't exist yet so no